Dockerfile

```diff
@@ -1,24 +1,25 @@
 # SPDX-License-Identifier: Apache-2.0
+# Copyright 2019-present Open Networking Foundation
 # Copyright (c) 2019 Intel Corporation
 
 # Multi-stage Dockerfile
-
 ARG BASE_OS=ubuntu:18.04
 ARG RUN_BASE=runtime
 
 ## Stage build: kitchen sink stage for compiling dependencies and CP/DP bins
 FROM $BASE_OS as build
 ARG CPUS
+ARG RTE_MACHINE=native
+ARG EXTRA_CFLAGS='-DUSE_AF_PACKET -ggdb -O2'
 
 WORKDIR /ngic-rtc
+SHELL ["/bin/bash", "-c"]
```

Comment:
Adding this to runtime stage will help remove `bash -c` from rest of the stages

Reply:
In my test, it only affects to the same stage where it is.